metadata-app/src/components/MetadataForm/index.js

```diff
@@ -131,8 +131,7 @@ const MetadataForm = (props) => {
             // update or create dataset:
             if (curDatasetId) {
               Api.updateDataset(curDatasetId, values, apiUrl, apiKey)
-                .then((res) => {
-                  setFormValues(Object.assign({}, res, { description: res.notes }));
```

Comment:
Why do we need to remove this? I thought it's a part of edit mode implementation?